### ISSUE TITLE
Can't inspect some textview

### ISSUE THREAD

**Issue by nnb-phil** [appium/appium#18939]
### Do I have the most recent component updates?

- [X] I use the most recent available driver/plugin and server versions

### Is the component officially supported by the Appium team?

- [X] I have verified the component repository is present under the Appium organization in GitHub

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Can't inspect some textview
![image](https://github.com/appium/appium/assets/141297581/f7e3d577-0562-4198-8d94-8c6c51705b0b)

### Expected Behavior

can get textview

### Minimal Reproducible Example

Can't inspect some textview

### Environment

- Operating system: mac os
- Appium server version (output of `appium --version`):  v2.0.1
- Appium driver(s) and their version(s):  android 11
- Appium plugin(s) and their version(s): 
- Node.js version (output of `node --version`): v16.15.1
- `npm` version (output of `npm --version`): v9.8.1
- Last component(s) version which did _not_ exhibit the problem:
- Platform and version under test:
- Real device or emulator/simulator:

### Futher Information

can get textview

**Comment by DengSanlun** [comment]
### Same issue, anyone can help

#### all TextView that inside ViewGroup is not reachable, with both old appium 1.x and appium2, using latest appium inspector. seems it never reach the deep elements, even try set snapshotMaxDepth larger

**Comment by KazuCocoa** [comment]
I wondered if you could set content description to the elements inside the view group. If the element was an image with image view (so, as AccessibilityNodeInfo in Android, it is a single "image view" element), probably the element cannot split for Android though.
https://discuss.appium.io/ could also help to ask such questions.

**Comment by nnb-phil** [comment]
I can't change anything to these apps, but pretty sure that the imageView is only the icon, and below text not included in the imageView

**Comment by KazuCocoa** [comment]
Then, what I could suggest is https://github.com/appium/appium-uiautomator2-driver#elements-cannot-be-found .

If still not work, perhaps Appium cannot detect it since it might indicate AccessibilityNodeInfo did not detect the element details as Appium depends on [AccessibilityNodeInfo](https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo).
Usually adding content description explicitly, and tweaking the view structure helps to make it detectable by Android (e.g. by voice over), then it helps AccessibilityNodeInfo. but maybe your case cannot do such thing.

**Comment by nnb-phil** [comment]
![image](https://github.com/appium/appium/assets/141297581/103c2db9-d250-466e-8cde-c96f868d6d1d)
![image](https://github.com/appium/appium/assets/141297581/4c93b91a-3010-4496-87f3-3417b2c4af8f)
![image](https://github.com/appium/appium/assets/141297581/2056c196-4250-448c-a94d-8146e86c26a6)

I have tried all related setting, nothing changed. from the screenshot you can see that it just looks like the textView is missing

**Comment by KazuCocoa** [comment]
They are settings api

**Comment by nnb-phil** [comment]
> They are settings api

Can you explain more.

**Comment by DengSanlun** [comment]
i guess some customised class can't be inspected, then auto changed to 'android.view.View'. is there any way to get the raw data before decode?

**Comment by KazuCocoa** [comment]
https://appium.io/docs/en/2.0/guides/settings/#initializing-settings-via-capabilities might help for your checking @nnb-phil 
 The page also explains the settings api further.


> i guess some customised class can't be inspected, then auto changed to 'android.view.View'. is there any way to get the raw data before decode?

Perhaps it is how Android works, then, we have no much methods can try. My previous comment https://github.com/appium/appium/issues/18939#issuecomment-1663396836 is possible methods we have (and we've been aware of so far) @DengSanlun

**Comment by jlipps** [comment]
Yeah, apart from the `ignoreUnimportantViews` setting, there's probably nothing that can be done on the UiAutomator2/Appium side. You'd need to work with the app developer to fix their accessibility settings or the way they're composing the UI element.

**Comment by JiuJiang-w** [comment]
I also encountered the same problem, except for the lack of textview, there is also a lack of EidtText tags
